macros/mock_builders.sql

```diff
@@ -66,7 +66,15 @@
     {% set input_values_sql %}
       {% set node_sql = dbt_unit_testing.build_node_sql(model_node, options.use_database_models) %}
         select * from ({{ input_values_sql }}) as m1
-        left join (select {{ missing_columns | join (",")}}
+        left join (select 
+        {%- for column in missing_columns %}
+          {%- if options.quoted_columns %}
+            "{{ column }}"
+          {%- else %}
+            {{ column }}
+          {%- endif %}
```

Comment:
I'm new here, and was browsing the open PRs, but this seems pretty snowflake specific -- does it make sense to use the [quote_identifier](https://github.com/EqualExperts/dbt-unit-testing/blob/c997099c3c747115e8ec6a2a8d658b2d33e7cb85/macros/utils.sql#L131C10-L131C26) macro here?

It sounds like capitalized identifiers are valid, and that would dispatch to [snowflake__quote_identifier](https://github.com/EqualExperts/dbt-unit-testing/blob/c997099c3c747115e8ec6a2a8d658b2d33e7cb85/macros/utils.sql#L151) which surrounds the column name in double quotes and capitalizes it: 
```
return('"' ~ identifier | upper ~ '"')
``` 
This would let users of the library override the behaviour in their projects if needed.

Reply:
Thanks for the comment Roger. You make a good point.
Coincidentally I was having the exact same conversation at a hackathon last night.

I will be updating this to a more generic/technology agnostic implementation.